Makefile

```diff
@@ -126,19 +128,21 @@ help:
 	@echo "    unicharset       Create unicharset"
 	@echo "    charfreq         Show character histogram"
 	@echo "    lists            Create lists of lstmf filenames for training and eval"
-	@echo "    training         Start training"
+	@echo "    training         Start training (i.e. create .checkpoint files)"
```

Comment:
```suggestion
	@echo "    training         Start training"
```

Reply:
I'd prefer a less technical description here and avoid details which might change any time.

Reply:
Like I [said above](https://github.com/tesseract-ocr/tesstrain/pull/377#discussion_r1516591680), the description of `traineddata` already mentions .checkpoint files, too. And so does `evaluation`. So as a matter of clarity and consistency, it should be mentioned here as well.

Since this entire repo is just a small makefile wrapper around the actual training tools, which are only properly documented in tessdoc _along these technical details_, I prefer keeping it the way it is.